packages/addon-dev/src/rollup-hbs-plugin.ts

```diff
@@ -63,6 +63,7 @@ export default function rollupHbsPlugin({
       let syntheticId = needsSyntheticComponentJS(source, resolution.id);
       if (syntheticId) {
         this.addWatchFile(source);
+        this.addWatchFile(syntheticId);
```

Comment:
this change did not work @ef4 .
I think transform does have its own specific cache.
https://rollupjs.org/plugin-development/#transform